Apeswap-Bots/New-Pair-Creation/src/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { APEFACTORY_ABI } from "./constants";
+
+const { CREATE_PAIR_FUNCTION } = APEFACTORY_ABI;
+
+// finding type definition
+type newPairFindingType = {
+  tokenAAddress: string;
+  tokenBAddress: string;
+};
+
+type newPairParamsType = {
+  functionSig: string;
+};
+
+const providerParams: newPairParamsType = {
+  functionSig: CREATE_PAIR_FUNCTION,
+};
+
+const createFinding = (findingMetadata: newPairFindingType, functionAbi: string): Finding => {
+  const findingResult = {
+    name: "New pair creation on ApeFactory contract",
+    description: `${functionAbi} call detected on ApeFactory contract upon creation of new tradable pairs`,
```

Comment:
```suggestion
    description: `${functionAbi} call detected on ApeFactory contract `,
```

Besides changing the functionAbi to just the function name, I would make this shorter as it's already descriptive